lib/active_record/mass_assignment_security/core.rb

```diff
@@ -1,6 +1,12 @@
 module ActiveRecord
   module MassAssignmentSecurity
     module Core
+      ALLOWED_CLASSES = [
+        'ActionController::Parameters',
+        'ActiveSupport::HashWithIndifferentAccess',
+        'Audited::AuditChanges',
+        'Hash',
+      ].to_set.freeze
```

Comment:
This is why I'd really like to get rid of this gem. Maintaining compatibility means a list of allowed classes to have their attributes assigned to now. If we didn't have a ticket to remove it I'd look for a better solution, but this works for now.